src/bin/gpfdist/regress/input/exttab1.source

```diff
@@ -607,7 +607,7 @@ CREATE WRITABLE EXTERNAL TABLE writable_external_table_with_serial_column_type (
     LOCATION ('gpfdist://@hostname@:7070/external_table_with_serial_column_type.csv')
     FORMAT 'CSV' (DELIMITER ',' NULL '' ESCAPE '"' QUOTE '"') ENCODING 'UTF8' DISTRIBUTED RANDOMLY;
 
-CREATE READABLE EXTERNAL TABLE readable_external_table_with_serial_column_type(LIKE writable_external_table_with_serial_column_type)
+CREATE READABLE EXTERNAL TABLE readable_external_table_with_serial_column_type(a INT, b BIGINT)
```

Comment:
> It would be great to add the case with LIKE to show that LIKE is also covered by the patch.

LIKE is covered by the patch, but here the readable table has different column types (BIGINT instead of BIGSERIAL). Should I create an additional table with BIGINT so that I can inherit it with LIKE?

Reply:
Sorry, I still can not see the test which tries to create a table with LIKE.

Reply:
The test was already there before this PR, [here](https://github.com/arenadata/gpdb/blob/adb-6.x-dev/src/bin/gpfdist/regress/input/exttab1.source#L610). I just reverted the changes back to the original since LIKE seems to work now.

Reply:
I still suggest making a test with LIKE and readable table based on some source table with serial column to demonstrate the behavior in this case.